_component/countdown-timer/content/_notes.md

```diff
@@ -0,0 +1,6 @@
+The Countdown Timer is an accessible utility component designed to work with a minimal amount of setup and HTML markup. It can display intervals of years, weeks, days, hours, minutes, and seconds, and supports many options for controlling display and behavior. It also supports callback functions on timer creation, countdown end, and on each timer update (more or less every second), which can be useful for triggering behavior in other components on the page. <a href="https://github.com/10up/component-countdown-timer#api">Read more about the component’s API here</a>.
```

Comment:
Same comment about linking to the readme, let's pull all relevant documentation in here. More like what we're doing with animate https://github.com/10up/component-animate/blob/master/README.md